webv2/helpers/helpers.go

```diff
@@ -37,7 +37,7 @@ const (
 	GOOGLE_SQL_DIALECT     = "Google Standard SQL"
 )
 
-const metadataDbName string = "harbourbridge_metadata"
+const metadataDbName string = "spannermigrationtool_metadata"
```

Comment:
nit: will it be better to rename the table as spanner_migration_tool_metadata for better readability?

Reply:
`spanner_migration_tool_metadata` exceeds the allowed database name length in Spanner.